examples/super_convergence/super_convergence.ipynb

```diff
@@ -0,0 +1,2061 @@
+{
```

Comment:
<p><code>lr=0.1</code> is used in line 5 and wondering how this value is found.</p><p>Did you set this <code>lr</code> from some findings from <code>find_lr</code> call in the previous cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/objax/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Yes from the figure approximately at 0.1 the loss started decreasing.</p>

Reply:
Is there a systematic way to get `lr` instead of eyeballing?
If not, would you add a comment how to select `lr` in practice?

Reply:
Ok, I'll do it.

Reply:
Hi @sathish-a, just a kind reminder. Please let us know if there is anything that we could help.

Reply:
@kihyuks thanks for the reminder. I added it.

Reply:
thanks!